import { AxiosError } from 'axios';
export declare const handleErrorResponse: (error: AxiosError) => void;
